src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,179 @@
+     */
+    function calculationIntervalSeconds() external view returns (uint32);
+
+    /// @notice The maximum amount of time that a range payment can end in the future
+    function MAX_PAYMENT_DURATION() external view returns (uint32);
+
+    /// @notice The lower bound for the start of a range payment
+    function LOWER_BOUND_START_RANGE() external view returns (uint32);
+
+    /// @notice Delay in timestamp before a posted root can be claimed against
+    function activationDelay() external view returns (uint32);
+
+    /// @notice Mapping: account => the address of the entity to which new payments are directed on behalf of the account
+    function recipientOf(address account) external view returns (address);
+
+    /// @notice Mapping: recipient => token => total amount claimed
+    function cumulativeClaimed(address recipient, IERC20 token) external view returns (uint256);
+
+    /// @notice the commission for all operators across all avss
+    function globalOperatorCommissionBips() external view returns (uint16);
+
+    /// @notice returns the hash of the leaf
+    function calculateLeafHash(ClaimsTreeMerkleLeaf calldata leaf) external view returns (bytes32);
+
+    /// @notice returns 'true' if the claim would currently pass the check in `processClaims`
+    function checkClaim(PaymentMerkleClaim calldata claim) external view returns (bool);
+
+    /// EXTERNAL FUNCTIONS ///
+
+    /**
+     * @notice Initializes the contract
+     * @param initialOwner The address of the initial owner of the contract
+     * @param _paymentUpdater The address of the initial `paymentUpdater`
+     * @param _activationDelay Delay in timestamp before a posted root can be claimed against
+     * @param _globalCommissionBips The commission for all operators across all avss
+     * @dev Only callable once
+     */
+    function initialize(address initialOwner, address _paymentUpdater, uint32 _activationDelay, uint16 _globalCommissionBips) external;
+    
+    /**
+     * @notice Creates a new range payment on behalf of an AVS, to be split amongst the
+     * set of stakers delegated to operators who are registered to the `avs`
+     * @param rangePayment The range payment being created
+     * @dev Expected to be called by the ServiceManager of the AVS on behalf of which the payment is being made
+     * @dev The duration of the `rangePayment` cannot exceed `MAX_PAYMENT_DURATION`
+     * @dev The tokens are sent to the `claimingManager` contract
+     * @dev This function will revert if the `rangePayment` is malformed,
+     * e.g. if the `strategies` and `weights` arrays are of non-equal lengths
+     */
+    function payForRange(RangePayment calldata rangePayment) external;
+
+    /**
+     * @notice similar to `payForRange` except the payment is split amongst *all* stakers
+     * rather than just those delegated to operators who are registered to a single avs
+     */
+    function payAllForRange(RangePayment calldata rangePayment) external;
```

Comment:
Does this emit a range payment with `avs = address(0)`?

Reply:
you're talking about the event, right?
`avs` is no longer a field in the RangePayment struct
Perhaps we should have a 2nd event here?